src/Writers/LaTeXWriter.jl

```diff
@@ -404,10 +407,11 @@ function latexinline(io::IO, md::Markdown.Italic)
 end
 
 function latexinline(io::IO, md::Markdown.Image)
+    doc = task_local_storage(:doc)
     wrapblock(io, "figure") do
         _println(io, "\\centering")
         wrapinline(io, "includegraphics") do
-            _print(io, md.url)
+            _print(io, joinpath(doc.user.root, doc.user.build, md.url))
```

Comment:
Thanks for the review. Yes, copying the whole build over will work. I've been wondering what option to choose in fixing this. If you think copying the whole `build` directory is an acceptable solution, then things are quite simple.